### PR TITLE
Add clone date method

### PR DESCRIPTION
**Pull request by kborucinski**
Add clone date method

### CODE REVIEW

File: date/clone.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "clone",
+  "description": "Returns the cloned date",
```
Comment (przemyslawzalewski) [inline date/clone.json]: Consider this:

```suggestion
  "description": "Clones the given Date object.",
```

File: date/clone.test.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/* eslint-env jest */
+// @ts-ignore ambiguous import
+import clone from "./clone.ts";
+
+describe("clone", () => {
+  it("should returns cloned date", () => {
```
Comment (przemyslawzalewski) [inline date/clone.test.ts]: Please simplify the test case with `toBe` and `equals` comparison.

Comment (marekrozmus) [inline date/clone.test.ts]: returns => return

File: README.md [hunk 3]
```diff
@@ -595,6 +595,26 @@ Runs the given tasks in a sequence.
...
+```typescript
+(date: Date) => Date
+```
+<!-- prettier-ignore-end -->
+
+##### Examples
+
+<!-- prettier-ignore-start -->
+```javascript
+clone(new Date('2019-04-24T13:54:33.232Z')); // ⇒ Wed Apr 24 2019 15:54:33 GMT+0200 (czas środkowoeuropejski letni)
```
Comment (marekrozmus) [inline README.md]: change `(czas środkowoeuropejski letni)` to english

File: README.md [hunk 4]
```diff
@@ -595,6 +595,27 @@ Runs the given tasks in a sequence.
...
+(date: Date) => Date
+```
+<!-- prettier-ignore-end -->
+
+##### Examples
+
+<!-- prettier-ignore-start -->
+```javascript
+const date = new new Date('2019-04-24T13:54:33.232Z');
+clone(date) !== date; // ⇒ true
```
Comment (przemyslawzalewski) [inline README.md]: Please showcase value equality.

```suggestion
const cloned = clone(date);
cloned  !== date && cloned.valueOf() === date.valueOf(); // ⇒ true
```

File: date/clone.test.ts [hunk 5]
```diff
@@ -0,0 +1,11 @@
+/* eslint-env jest */
+// @ts-ignore ambiguous import
+import clone from "./clone.ts";
+
+describe("clone", () => {
+  it("should return cloned date", () => {
+    const date = new Date("2019-04-24T13:54:33.232Z");
+
+    expect(clone(date)).not.toBe(date);
```
Comment (przemyslawzalewski) [inline date/clone.test.ts]: Please assert value equality as well.

```suggestion
    const cloned = clone(date);
    expect(cloned).not.toBe(date);
    expect(cloned).toEqual(date);
```

**Pull request merged**